dozer-ingestion/src/connectors/mysql/binlog.rs

```diff
@@ -139,8 +139,24 @@ impl BinlogIngestor<'_, '_, '_, '_, '_> {
         let mut table_cache = BinlogTableCache::new(self.tables);
 
         'binlog_read: while let Some(event) = self.binlog_stream.as_mut().unwrap().next().await {
+            match self.local_stop_position {
```

Comment:
Moving this block to above `self.next_position.position = binlog_event.header().log_pos().into();` changes the behavior. Is that intended?

Reply:
Some unrelated strange test failures were happening when I added `if is_artificial { continue; }`, and they when away when I moved this block from the bottom to the top. It's a minor change and it's still correct. I am not going to try to figure out what's going on because that's valuable time that I should spend elsewhere.

Reply:
:+1: , just wanted to make sure it was intentional